packages/wallet/src/views/Wallet.jsx

```diff
@@ -36,24 +36,24 @@ const Wallet = () => {
         </div>
         <div className='lg:px-64 md:px-12 px-12 py-20'>
           <div className='font-heading lg:text-7xl text-5xl text-primary lg:py-5'>
-            Portal.
+            Welcome to Kernel.
           </div>
-          <div className='font-secondary text-lg text-kernel pt-12'>
+          <div className='font-secondary text-3xl text-kernel pt-12'>
             <Link to='/register'>
               <span className='pr-2 before:block before:absolute before:-inset-1 before:-skew-y-3 before:bg-kernel-highlight relative inline-block cursor-pointer'>
                 <span className='relative text-primary'>
                   <span className='underline decoration-dotted'>
-                    Start your adventure
+                    Setup your account
                   </span>
                 </span>
               </span>
             </Link>
-            &nbsp;or&nbsp;
+            &nbsp;then&nbsp;
```

Comment:
This makes it sound sequential, when they are really two separate options: create an account, or log in with one you have already created...